doc/manual/generate-builtins.nix

```diff
@@ -1,16 +1,20 @@
-with builtins;
-with import ./utils.nix;
+builtinsDump:
```

Comment:
Not sure about the argument name. It could as well be `builtinsInfo`. I changed it because it was really hard to discern from the actual `builtins`.